doc/page-features.h

```diff
@@ -151,4 +151,18 @@ foo.userData<Pod>().i;  // 42
 An interface implementation can expose a structure containing some user data. This user data is
 preserved when the structure is copied. Thus it can be made available in a different interface.
 
+\section ServerImplementation Server inside C++ Implementation
+When user doesn't want to explicitly use \c IPC.enabled: \c true flag inside qml UI code, and wants to create server inside C++ Implementation, one can do the following
```

Comment:
Sounds a bit clumsy... maybe...
"Instead of enabling IPC in the QML UI code (with \c IPC.enabled: \c true), it is possible to register an IPC server in the C++ implementation. The following code snippet shows, how this can be achieved:"

Reply:
I agree with Bernd, but with "C++ code" instead of "C++ implementation"